ad/views.py

```diff
@@ -122,20 +122,10 @@ class AdDeleteView(LoginRequiredMixin, DeleteView, ModelMappingMixin):
     context_object_name = 'ad'
 
     def get_queryset(self):
-        # Get the model based on ad type
         model = self.get_model()
         return model.objects.all()
```

Comment:
This get_queryset method is not needed here right ? The same exists in the mixin

Reply:
If i remove this method code gives error